.eslintrc.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  extends: ["airbnb", "prettier", "prettier/react"],
+  parser: "babel-eslint",
+  plugins: ["prettier"],
+  rules: {
+    "prettier/prettier": ["error"],
+    // Allow JSX in .js file extension as well as default .jsx
+    "react/jsx-filename-extension": ["error", { extensions: [".js", ".jsx"] }],
+    // next.js automatically has React in scope, so turn off requiring React
+    "react/react-in-jsx-scope": "off"
```

Comment:
Is this true for projects that use Next.js? e.g. the project I'm currently working on? 

I'd love to remove the redundancy 💃

Reply:
`import React from 'react'` is automatically injected when using JSX, https://github.com/vslinko/babel-plugin-react-require is included